sdk/core/azure-core/README.md

```diff
@@ -14,7 +14,7 @@ Typically, you will not need to install or specifically depend on Azure Core, in
 <dependency>
   <groupId>com.azure</groupId>
   <artifactId>azure-core</artifactId>
-  <version>1.0.0-preview.3</version>
+  <version>1.0.0-preview.5</version>
```

Comment:
this should be preview.4

Reply:
We are releasing `1.0.0-preview.5`  next week for azure-core, thus README should also point to `1.0.0-preview.5`